_api-reference/ingest-apis/processors-reference/append.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Append
+parent: Ingest processors 
+grand_parent: Ingest APIs
+nav_order: 10
+---
+
+# Append
+
+The append ingest processor enriches incoming data during the ingestion process by appending additional fields or values to each document. The append processor operates on a per-dcoument basis, meaning it processes each incoming document individually. Learn how to use the append processor in your data processing workflows in the following documentation. 
```

Comment:
I think this tell nothing about append processor. 
1. `incoming data during the ingestion process` this is same for all processors. 
2. `The append processor operates on a per-dcoument basis, meaning it processes each incoming document individually.` same for all processors.